db/migrate/20171113153954_create_transactions.rb

```diff
@@ -0,0 +1,11 @@
+class CreateTransactions < ActiveRecord::Migration[5.1]
+  def change
+    create_table :transactions do |t|
+    	t.date :transaction_date
+    	t.integer :transaction_score
```

Comment:
I think the prefix `:transaction_` here is unnecessary, since calling it would look something like `@transaction.transaction_score`. It would read better if it was just `@transaction.score` IMO